documentation/README.md

```diff
@@ -1523,6 +1525,26 @@ nth(1, [1, 2, 3]); // => 2
 </sup></div>
 
 
+### omit
+
+Creates a copy of the `object` without the given `props`.
+
+```js
+const omit = require('1-liners/omit');
+
+const object = {foo: 1, bar: 2, baz: 3};
+
+omit(['foo', 'baz'], object);  // => {bar: 2}
+object;                        // => {foo: 1, bar: 2, baz: 3}
```

Comment:
Do you mean the line:
> Creates a copy of the `object` without the given `props`. 

Because I just changed the name of the method from `drop` to `omit` it was already that description see [here](https://github.com/1-liners/1-liners/tree/master/documentation#drop).

I can create a better one if needed?